packages/agent/package.json

```diff
@@ -72,6 +72,7 @@
     "@noble/ciphers": "0.4.1",
     "@scure/bip39": "1.2.2",
     "@tbd54566975/dwn-sdk-js": "0.4.0",
+    "@web5/api": "^0.9.4",
```

Comment:
Since `@web5/api` imports `@web5/agent` for an abstraction over dealing with eh `dwn-sdk-js` directly, I'd like to avoid the circular dependency here.

Maybe we could make the `dwnEndpoints` optional as you have it and pass the endpoints down from the `connect()` function through the `Web5UserAgent.initialize()` method and into `HdIdentityVault.initialize()` and then optionally add the service endpoint to the document if it exists.

Reply:
ACK. Will work on that today.

Reply:
@LiranCohen Reverted circular dependency and added code to pass `dwnEndpoints` down from `Web5.connect` to `HdIdentityVault.initialize`. Have not tested. Will be doing that today/tomorrow.